bing_bert/deepspeed_train.py

```diff
@@ -32,15 +33,16 @@
 all_step_time = 0.0
 
 
-def checkpoint_model(PATH, ckpt_id, model, epoch, last_global_step,
+def checkpoint_model(PATH, ckpt_id, model, optimizer, epoch, last_global_step,

```

Comment:
Passing optimizer should not be required since this code previously worked without it. Were you seeing some issue that required this change?